server/src/main/java/io/spine/server/delivery/Delivery.java

```diff
@@ -436,6 +441,27 @@ public Optional<DeliveryStats> deliverMessagesFrom(ShardIndex index) {
         return Optional.of(stats);
     }
 
+    /**
+     * Notifies the {@code DeliveryMonitor} about the technical failure and executes
+     * the failure handler {@code Action}.
+     */
+    private Optional<DeliveryStats> handleTechPickUpFailure(ShardIndex shard, RuntimeException e) {
```

Comment:
Let's shorten to `onRuntimeFailure`.

I don't think that saying "technical <something>" really explains something. It's rather an "unexpected runtime <something>". This goes for both the documentation of this method, and for the `TechFailure` name.